src/components/Form.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Form = () => (
+  <div>
+    <h2>ADD NEW BOOK</h2>
+    <input placeholder="Book title" />
+    <input placeholder="Author" />
+    <button type="submit">ADD BOOK</button>
```

Comment:
- [ ] This section should render a form with two inputs and a button to submit the form (the button is just a placeholder for now), kindly wrap the inputs and submit button with a form tag so your form can be submitted in the future when adding the functionality

Reply:
This issue is well resolved! ✔️